spec/jobs/ignored_users_summary_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+require_dependency 'jobs/scheduled/ignored_users_summary'
+
+describe Jobs::IgnoredUsersSummary do
+  before do
+    SiteSetting.ignore_user_enabled = true
+    SiteSetting.ignored_users_count_message_threshold = 1
+    SiteSetting.ignored_users_message_gap_days = 365
+  end
+
+  subject { Jobs::IgnoredUsersSummary.new.execute({}) }
+
+  context "when no ignored users" do
+    it "does nothing" do
+      subject
+      expect(Post.count).to eq(0)
+    end
+  end
+
+  context "when some ignored users exist" do
+    let(:tarek) { Fabricate(:user, username: "tarek") }
+    let(:matt) { Fabricate(:user, username: "matt") }
+    let(:john) { Fabricate(:user, username: "john") }
+
+    before do
+      Fabricate(:ignored_user, user_id: tarek.id, ignored_user_id: matt.id)
```

Comment:
I *think* you can be clever a do this instead

```suggestion
      Fabricate(:ignored_user, user: tarek, ignored_user: matt)
```